wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/DetachedActivity.java

```diff
@@ -90,6 +90,8 @@ private boolean validated_device(){
         if(devices.getDeviceCount() == 0)
             return false;
         Device device = devices.createDevice(0);
+        if(!device.supportsInfo(CameraInfo.RECOMMENDED_FIRMWARE_VERSION))
+            return true;
```

Comment:
Breaking here will cause a possible FW field not reaching UI thread in  "textView.setText("T..." code. is it expected?

Reply:
That means that I can tell if the FW is valid or not and that I will try to use this device anyways.
The message is shown only for invalid devices